conf/solr/8.8.1/schema.xml

```diff
@@ -359,7 +359,7 @@
     <field name="responseRate" type="text_en" multiValued="false" stored="true" indexed="true"/>
     <field name="samplingErrorEstimates" type="text_en" multiValued="false" stored="true" indexed="true"/>
     <field name="samplingProcedure" type="text_en" multiValued="false" stored="true" indexed="true"/>
-    <field name="series" type="text_en" multiValued="false" stored="true" indexed="true"/>
+    <field name="series" type="text_en" multiValued="true" stored="true" indexed="true"/>
```

Comment:
I think we want the subfields of "series" to be multivalued as well. Please check out this diff:

```
-    <field name="series" type="text_en" multiValued="false" stored="true" indexed="true"/>
-    <field name="seriesInformation" type="text_en" multiValued="false" stored="true" indexed="true"/>
-    <field name="seriesName" type="text_en" multiValued="false" stored="true" indexed="true"/>
+    <field name="series" type="text_en" multiValued="true" stored="true" indexed="true"/>
+    <field name="seriesInformation" type="text_en" multiValued="true" stored="true" indexed="true"/>
+    <field name="seriesName" type="text_en" multiValued="true" stored="true" indexed="true"/>
```

Reply:
I added the subfields of series in 4509dea.

Reply:
Or wouldn't it make more sense to change Series to a complex field, such as Author? I guess this would mean:

series
allowmultiples: TRUE

seriesName
allowmultiples: FALSE

seriesInformation
allowmultiples: FALSE